doc/packages.md

```diff
@@ -70,12 +70,15 @@ On each commit, Circle CI automatically compiles all supported packages with Gop
 | go              |                       |                                   |
 | -- ast          | yes                   |                                   |
 | -- build        | no                    |                                   |
+| -- constant     | yes                   |                                   |
 | -- doc          | yes                   |                                   |
 | -- format       | yes                   |                                   |
+| -- importer     | yes                   | (TODO: unconfirmed, no tests)     |
```

Comment:
Well, just looking at [its source](http://gotools.org/go/importer?tests=) is all that was needed.

It imports `go/types`, `runtime`, `go/internal/gcimporter` and `go/internal/gccgoimporter`.

Importing `go/types` is enough to make it not work, since `go/types` tests fail.